library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/QuiltResourcePackProfile.java

```diff
@@ -39,4 +40,18 @@ public interface QuiltResourcePackProfile {
 	default @NotNull ResourcePackActivationType getActivationType() {
 		return ResourcePackActivationType.NORMAL;
 	}
+
+	static ResourcePackProfile.ResourcePackFactory identityFactory(ResourcePack pack) {
```

Comment:
Why is this in API?

Reply:
I moved it to API after noticing that the testmods need it. Vanilla does exactly the same thing, but their implementation `BuiltinResourcePackProvider#method_52435(ResourcePack)`is `protected`